app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java

```diff
@@ -22,16 +24,27 @@ public void clickedImage(final ImageView imageView) {
         imageView.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
+                PackageManager pkManager = view.getContext().getPackageManager();
                 String url = "";
                 if (imageView.getTag().toString().equals(view.getContext().getString(R.string.linkedin))) {
                     url = speaker.getLinkedin();
-
                 } else if (imageView.getTag().toString().equals(view.getContext().getString(R.string.twitter))) {
                     url = speaker.getTwitter();
-
+                    String idname = url.substring(url.lastIndexOf('/') + 1);
+                    try {
+                        PackageInfo pkgInfo = pkManager.getPackageInfo("com.twitter.android", 0);
```

Comment:
We should probably only send the http://twitter intent, and let the user chose his default perferene to open it. 

I prefer to open twitter in website for example, even when I have the app